docs/detections/alerts-view-details.asciidoc

```diff
@@ -58,18 +58,11 @@ The Insights section provides the following details:
 
 * *Cases related to the alert* - Shows the total number and names of cases to which the alert has been added. Click a case's name to open its details.
 * *Alerts related by source event* - Shows the ten most recent alerts created by the same source event. This can help you find alerts with a shared origin and provide more context about the source event. Click the *Investigate in timeline* button to examine related alerts in Timeline.
-* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same Session ID, which is a unique ID for tracking a given Linux session.
-+
-beta::[]
-+
-NOTE: This feature requires a https://www.elastic.co/pricing[Platinum or Enterprise subscription]. In addition, the *Include session data* setting must be enabled on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
 
+If you have a https://www.elastic.co/pricing[Platinum or Enterprise subscription], these details are also included:
+
+* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting on your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Comment:
Prepositions are tricky, but I like to thing of config settings are "within" a window or inside, hence `in`. But it's not crucial.  
```suggestion
* *Alerts related by session ID* - Shows the ten most recent alerts generated during the same <<session-view, session>>. These alerts share the same session ID, which is a unique ID for tracking a given Linux session. To use this feature, you must enable the *Include session data* setting in your {elastic-defend} integration policy. Refer to <<enable-session-view, Enable Session View data>> for more information.
```

Reply:
I think both work, but "in" sounds a bit more correct when I think about where the setting is located _within_ the policy.